etc/orthography.tsv

```diff
@@ -0,0 +1,88 @@
+Grapheme	IPA	EXAMPLES
+^	NULL	0
+$	NULL	0
+-	NULL	wai-oot;i-thlung-a;i-thlung-a;i-hling-a;i-hling-a
+´	NULL	pu-kwā´-num;ka-tit´-too;in´-to-op;noo-ka-tam´-tha;is´-hail-o-wih
+’	NULL	shi-li-kit-tim-win-’go;ni-’ko’k-a-hik;ni-’ko’k-a-hik;tsoo-wan-kit-tim-win-’go;spl-’kein
+gw	gʷ	duha-ho-lo-gwodi;tsel´-e-de-gwus;stō-le-gwan;goo-gwilla;goo-gwilla
+hw	hʷ	hahwi-e;koos-hwa;koos-hwa;koos-hwa;kla-le-tla-hwlth
+lh	ʎ	til-koolh-im-yoot;til-koolh-im-un-nah;slin-ealh-kum-ealh;kulh-lātik;nish-ai-ulh-slannē
+ll	ɬ	ikalla;ska-kullā;nis-tallis;nis-tallis;kwullā
+sh	ʃ	kash-kash;shin-shin-ashi;shin-shin-ashi;shish-ootum;kash-kash
+wh	wʰ	tsi-hi-a-whatl;kus-whun-ook;kus-whun-ook;hun-whun-ookwi;is´-whuh-sē-whult
+ōh	ō ʔ	slanai-ōhl;kōht
+^āh	ā ʔ	āh-alph
+^īh	ī ʔ	īha-kommitz;īh-ā;īh-ā;īhl-thlē;īh-kwīn
+hs$	ʔ s̩	thluhs;eiloohs;luhs;poot-umit-kwo-luhs
+tsh	tʃʰ	tloo-tshimun;sē-tshin-kooi;si-asē-tshuk-oap;klootsh-timtz;te-tshā
+wh$	w ʔ	hul-lawh;slawh;yoawh;ow-awh;īdawh
+zh$	z ʔ	kwai-el-izh
+āh$	ā ʔ	hun-nāh;un-nāh;ni-lāh;hāh;hāh
+ēh$	ē ʔ	kuh-kwēh;tshin-kat-ka-tēh
+ĕh$	ĕ ʔ	skil-a-kĕh
+īh$	ī χ	shīh;hut-līh;hut-līh;sam-tlīh
```

Comment:
Is this indeed a /χ/? All other long vowels + "h" in final position have a glottal stop.

Reply:
True, I'll check.